### PR TITLE
AUT for User can create an account

### PR DESCRIPTION
**Pull request by Joberg95**
_**As a user
In order to use this messaging service
I would like to be able to sign-up**_

Feature test and unit test to see if a user can create an account

### CODE REVIEW

**Review by LauraRe — APPROVED** [review]
Looks good!! Nice ponging ;)

File: db/schema.rb [hunk 1]
```diff
@@ -1,18 +1,7 @@
...
-# incrementally modify your database, and then regenerate this schema definition.
-#
-# Note that this schema.rb definition is the authoritative source for your
-# database schema. If you need to create the application database on another
-# system, you should be using db:schema:load, not running all the migrations
-# from scratch. The latter is a flawed and unsustainable approach (the more migrations
-# you'll amass, the slower it'll run and the greater likelihood for issues).
-#
-# It's strongly recommended that you check this file into your version control system.
+
```
Comment (faraznaeem) [inline db/schema.rb]: Never ever ever touch the schema file, delete the file, run the migrations again to generate it.

File: features/user_sign_up.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Feature: User sign-up
+
```
Comment (faraznaeem) [inline features/user_sign_up.feature]: Remove extra line

File: features/user_sign_up.feature [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    As a user
+    In order to use this messaging service
+    would like to to sign-up
```
Comment (faraznaeem) [inline features/user_sign_up.feature]: `I would like to sign-up`

File: features/user_sign_up.feature [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Scenario: User visits page to Sign up 
+    When I click on "Sign up" 
+    And I fill in "Name" with "Bob"
+    And I fill in "Email" with "bob@bobby.com"
+    And I fill in "Password" with "0443431157"
+    And I fill in "Password confirmation" with "0443431157"
+    And I click on "Create"
+    Then I should see "Welcome! You have signed up successfully."
+
+Scenario: User visits page and Signs up incorrectly [SadPath]
```
Comment (faraznaeem) [inline features/user_sign_up.feature]: What is this sad path checking for? Password or email?
There should be at least two sad paths one for each.

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: features/user_sign_up.feature (hunk 3 above)
Comment (Joberg95) [reply features/user_sign_up.feature]: wait what, i did change that before i submitted it wtf??

File: features/user_sign_up.feature (hunk 4 above)
Comment (aleciabenjamin) [reply features/user_sign_up.feature]: It is testing the Sad path for incorrect password only.  Incorrect email is handled by Chrome.

**Review by faraznaeem — APPROVED** [review]
Good job

**Pull request merged**